routing/router.go

```diff
@@ -2,6 +2,7 @@ package routing
 
 import (
 	"bytes"
+	goErrors "errors"
```

Comment:
wow didn't realize we were using `github.com/go-errors/errors`.

Reply:
Yeah I had forgot about ti, till I hit a weird compiler error. I think w/ the latest version of the std lib Go error package, we don't really gain anything by using  `github.com/go-errors/errors` any longer. So we may want to do a sweep to remove it.